pkgs/applications/science/biology/seqtk/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zlib
+, libdeflate
+, isa-l
+}:
+
+gccStdenv.mkDerivation rec {
```

First, you should put `gccStdenv` among the arguments.
Second, the best way should be setting `stdenv = gccStdenv` in all-packages.nix.

Third, this is now irrelevant because of `CC=cc` setting:

```suggestion
stdenv.mkDerivation rec {
```